scroll-animations-1/Overview.bs

```diff
@@ -1396,15 +1237,64 @@ spec: cssom-view-1; type: dfn;
 	should it return percentage progress through that range,
 	or time progress through that range?
 
+# Appendix B: Timeline Scope # {#timeline-scoping}
+
+	ISSUE: This section should move to CSS-ANIMATIONS-2.
+
+	This appendix introduces the 'timeline-scope' property,
+	which allows declaring a timeline name’s scope on an ancestor
+	of the timeline’s defining element.
+
+# Declaring a Named Timeline’s Scope: the 'timeline-scope' property ### {#timeline-scope-property}
+
+	<pre class="propdef">
+		Name: timeline-scope
+		Value: none | <<custom-ident>>#
+		Initial: none
+		Applies to: all elements
+		Inherited: no
+		Computed value: the keyword ''timeline-scope/none'' or a list of [=CSS identifiers=]
+		Animation type: not animatable
+	</pre>
+
+	This property declares the scope of the specified timeline names,
+	allowing a named timeline
+	(such as a [=named scroll progress timeline=] or [=named view progress timeline=])
+	to be referenced by elements outside the timeline-defining element’s subtree--
+	for example, by siblings, cousins, or ancestors.
+
+	Values have the following meanings:
+
+	<dl dfn-for="timeline-scope" dfn-type=value>
+		<dt><dfn>none</dfn>
+		<dd>
+			No change in name timeline scope.
+
+		<dt><dfn><<custom-ident>></dfn>
+		<dd>
+			Declares the name of a matching named timeline defined by a descendant
+			to be in scope for this element and its descendants.
+
+			If no such timeline exists,
+			or if more than one such timeline exists,
+			instead declares an [=inactive timeline=] with the specified name.
+
+			Note: This property has no effect if a matching named timeline
```

Comment:
Well, that would require also specifying that if you set timeline-scope on multiple ancestors, the closest one determines the scope. But actually as currently defined, we pull the scope all the way up to the highest scope with that name. I think this is fine? But in any case if we want to change that, we should open an issue to discuss. :)

Reply:
Well, this was certainly my intention all along that it only goes to the nearest ancestor with the name so that you could set a timeline-scope name for a subtree and not have to worry about an ancestor declaring the same name scope resulting in an ambiguous timeline selection. Apologies if this wasn't clear in the issue:

From https://github.com/w3c/csswg-drafts/issues/7759#issuecomment-1520406239
> The downside is that without the double handshake you would still be contaminating the ancestor timeline namespace with a timeline that was only intended to be used in its subtree - which a developer could avoid by making the subtree also an explicit timeline root for the name.

Reply:
OK, I updated the definition to say that, see new commit.

It does seem like if this is the direction we want to go, we might also want to add:
* a keyword that blocks all timeline names
* a keyword that only blocks timeline names defined locally on this element

So I'll open a new issue...

Reply:
Filed https://github.com/w3c/csswg-drafts/issues/8915